frontend/components/text-classifier/results/RecordTextClassification.vue

```diff
@@ -18,6 +18,9 @@
 <template>
   <div class="record" v-if="record">
     <div class="record--left">
+      <div class="record--image-area" v-if="isRecordContainsImage">
+        <img :src="metadata._image_url" alt="image of the record" />
```

Comment:
If the image is really big will it overflow the record area?

Reply:
If the image is too big, I put a max-height.